docs/data_access.rst

```diff
@@ -115,3 +115,24 @@ on the collection and the catalog that is used by it. This is not a problem caus
 standardization between catalogs of EO data.
 
 
+Handling large vector data
+#######################
+
+Handling large volumes of data is crucial for making decisions, improving processes and having an efficient solution. 
+However, if you attempt to execute a process on significantly large data, you would probably encounter a 'Request Entity Too Large' 
+error because your 'POST' request was too large for the system to handle. In order to avoid this error, you can upload
+your vector data to a public location(e.g., via Google Drive) and use it as an URL. The data stored in the URL can be loaded using 
+:py:meth:`~openeo.rest.connection.Connection.vectorcube_from_paths`
+
+The code-snippets shown below provides an example of how this can be achieved:
+
+.. code-block:: python
+
+    parcels = connection.vectorcube_from_paths(["https://artifactory.vgt.vito.be/testdata-public/parcels/32TPT.pq"],format="parquet")
+    datacube = connection.load_collection("SENTINEL1_GAMMA0_SENTINELHUB",
+                temporal_extent = ["2021-05-12",'2021-06-01'])
+    s1_cube = datacube.aggregate_spatial(geometries=parcels,
+                           reducer="mean")
```

Comment:
please try to follow the code style guidelines for example snippets in the docs: https://open-eo.github.io/openeo-python-client/best_practices.html#line-length-management

Reply:
I see you  updated it, but I think it should be more like this (following the [black code style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html)):
```python
    parcels = connection.vectorcube_from_paths(
        ["https://github.com/Open-EO/openeo-python-client/blob/master/tests/data/Selected_aoi.parquet"],
        format="parquet",
    )
    datacube = connection.load_collection(
        "SENTINEL2_L2A",
        ...
```

Reply:
Sorry I missed that again. Thank you for reminding